### PR TITLE
Updates irrelevant test

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
Addresses #11433 
##### ISSUE TYPE
 - Bug 

##### COMPONENT NAME
 - UI

### CODE REVIEW

**Comment by akus062381** [comment]
I don't think UI QE needs to test this one, it can be merged once a dev approves

File: awx/ui/src/components/LaunchButton/LaunchButton.test.js
Comment (keithjgrant) [inline awx/ui/src/components/LaunchButton/LaunchButton.test.js]: boolean props can just be set as flags (`<button type="submit" onClick={handleLaunch} disabled />`)

**Review by keithjgrant — APPROVED** [review]

**Pull request merged**
